app/src/vuex/modules/account.js

```diff
@@ -33,6 +33,7 @@ export default ({ apollo }) => {
         if (user) {
           commit(`userSignedIn`, true)
           commit(`setUserInformation`, user)
+          localStorage.setItem(`auth_token`, JSON.stringify(user.uid))
```

Comment:
missing remove on sign out

Reply:
Oh true!

Reply:
I should also refresh the token for ApolloLink right? Should I use this one (https://www.npmjs.com/package/apollo-link-token-refresh)?

Reply:
Ah no, forget that. I saw we have the LinkObservable in `apollo.js`. But should we use then some kind of refreshToken logic in there like [here](https://stackoverflow.com/questions/50965347/how-to-execute-an-async-fetch-request-and-then-retry-last-failed-request/51321068#51321068)?

Reply:
let's improve later. can we check if the token is already outdated before sending the request?

Reply:
This is what I'm about to test

Reply:
Are u sure we need to send the UID? Don't we need to send a verification token?

Reply:
The token can be anything. In this case, Firebase authentication system uses this JWT they call user "uid". We could also create our own, but it would be unnecessary complexity :shrug: 

Not sure though if I'm following you

Reply:
See here: https://firebase.google.com/docs/auth/admin/custom-claims#examples_and_use_cases
The UID is just a user ID anyone can copy. We need an oauth token which they call idToken

Reply:
OK, I see. I will add this tomorrow :wink:

Reply:
Thought it was actually the same but guess the idToken is a time-based randomly-generated one right? Like the 2FA

Reply:
2FA has nothing to do with it. But yes ^^